Annotation/Deprecated.php

```diff
@@ -43,6 +49,7 @@ public function __construct($options = null)
             $this->since = isset($options['since']) ? new DateTime($options['since']) : null;
             $this->description = $options['description'] ?? null;
             $this->hideInDoc = $options['hideInDoc'] ?? false;
+            $this->triggerDeprecation = $options['triggerDeprecation'] ?? null;
```

Comment:
I would not make this field nullable, but instead give him the default value `true` as it gets from the configuration. This way you can always work with a boolean value and you dont have to care about the additional null case.

Reply:
I made it nullable on purpose.
Default = null = use global behaviour from config yaml
true = always issue a deprecation error, ignoring the config yaml setting
false = never issue a deprecation error, ignoring the config yaml setting

without null I couldnt differentiate between an option given excplicitly vs. implicit defaults

Reply:
haaa, very good!